app/mailers/host_mailer.rb

```diff
@@ -7,7 +7,7 @@ class HostMailer < ApplicationMailer
   def summary(options = {})
     raise ::Foreman::Exception.new(N_("Must specify a valid user with email enabled")) unless (user = User.find(options[:user]))
     hosts = User.as user do
-      Host::Managed.authorized_as(user, :view_hosts, Host)
+      Host::Managed.authorized(:view_hosts)
```

Comment:
for the reviewer: I don't recall but I think avoiding class specification led to errors, because the auto generated class would be `Host::Managed`, while the permission resource type is `Host`. Thorough testing is needed. It may however also be a leftover from the past, I think there were changes after discovered hosts also use `Host` for their permissions.

Reply:
@tbrisker ? I believe we only check for `Host::Base`, but the check would be now done on `Host::Managed` that doesn't feel right :thinking:
Won't we need to add it? https://github.com/theforeman/foreman/blob/709cdf4fe777160e8bbf5033593ddf814ce2ac48/app/services/authorizer.rb#L29